src/main/java/uk/gov/companieshouse/itemhandler/kafka/OrdersKafkaConsumer.java

```diff
@@ -101,8 +101,9 @@ public void processOrderReceivedError(org.springframework.messaging.Message<Orde
             handleMessage(message);
         }
         else {
-            LOGGER.info(String.format("Pausing error consumer \"%1$s\" as error recovery offset '%2$d' reached.",
-                    ORDER_RECEIVED_GROUP_ERROR, errorRecoveryOffset));
+            Map<String, Object> logMap = LoggingUtils.createLogMap();
+            logMap.put(ORDER_RECEIVED_GROUP_ERROR, errorRecoveryOffset);
```

Comment:
CH Logging standards - the key name needs to be snake_case not kebab-case
e.g. 
error_recovery_offset
Also, if possible, can we log the topic name and offset of the message? errorRecovery Offset is a constant in this class, so will always be 0L